### ISSUE TITLE
Notification (Sound or visual)

### ISSUE THREAD

**Issue by Unders0und** [mjkaufer/Messer#81]
It would be nice if we could be able to know when we receive a message while the tab is minimized (or when we're using another tab of the terminal itself).

I don't know if it's possible to make a sound through the app, though
maybe the header of the terminal could show some sort of.. I don't know, star "*", just like when it happens when we let a command running and it finishes the task

well, I guess you got the idea

ah, yeah, I love this project, it's being extremely helpful to me o/ thanks a lot

**Comment by Unders0und** [comment]
@tomquirk really? lol
can I switch the terminal? I'm using the terminal that comes in with the Lubuntu and I don't know how to make it sound XD
and it doesn't display anything at the header when I switch tab or minimize it

**Comment by tomquirk** [comment]
yeah I'm honestly not sure 🤔 Maybe have a read around for how other people get notifications on Ubuntu

**Comment by voidcase** [comment]
@tomquirk I think a lot of people have the terminal bell disabled to avoid immediate mental break. It's also off by default in many terminals. I would suggest switching from using terminal bell to a package like [play-sound](https://www.npmjs.com/package/play-sound) or [audio-play](https://www.npmjs.com/package/audio-play).

**Comment by michondr** [comment]
on ubuntu you can produce notification by calling `notify-send "title" "body"`

![image](https://user-images.githubusercontent.com/29277166/87870136-ac8e1180-c9a5-11ea-85c1-bdbb38727fab.png)

not sure how this works on other distros (or win/mac) though
